Competition2017/tests/competition/subsystems/shooter/SideShooterTest.java

```diff
@@ -0,0 +1,92 @@
+package competition.subsystems.shooter;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import competition.subsystems.shooter.commands.ContinueShooterCommand;
+import competition.subsystems.shooter.commands.SetShooterSpeedCommand;
+import competition.subsystems.shooter.commands.StepShooterPowerCommand;
+import competition.subsystems.shooter.commands.StopShooterCommand;
+
+public class SideShooterTest extends ShooterTestBase {
+    
+     SideShooterSubsystem leftShooter;
+     SideShooterSubsystem rightShooter;
+    
+    @Before
+    @Override
+    public void setup() {
+        super.setup();
+        leftShooter = shooter.getLeftShooter();
+        rightShooter = shooter.getRightShooter();
+        
+    }
+    
+    @Test
+    public void testContinueShooterCommand(){
+        ContinueShooterCommand cmd = injector.getInstance(ContinueShooterCommand.class);
+        
+        cmd.setSide((leftShooter));
+        cmd.initialize();
+        cmd.execute();
+        cmd.isFinished();
+        
+        cmd.setSide(rightShooter);
+        cmd.initialize();
+        cmd.execute();
+        cmd.isFinished();
+        
+         verifyShooterPowers(shooter.getLeftShooter().getLauncherPower(), shooter.getRightShooter().getLauncherPower());;
+    }
+    
+    @Test
+    public void testStepShooterPowerCommand(){
+        StepShooterPowerCommand sspc = injector.getInstance(StepShooterPowerCommand.class);
+        sspc.setSide(leftShooter);
+        
+        double leftTemp = shooter.getLeftShooter().getLauncherPower();
+        
+        sspc.initialize();
+        sspc.execute();
+        sspc.isFinished();
+        
+        verifyShooterPowers(leftTemp + leftShooter.getPowerStep(), 0);
+    }
+    
+    @Test 
+    public void testStopShooterCommand(){
+        StopShooterCommand stop = injector.getInstance(StopShooterCommand.class);
+        stop.setSide(leftShooter);
+        
+        stop.initialize();
+        stop.execute();
+        stop.isFinished();
+        
+        verifyShooterPowers(0,0);
```

Comment:
Ahh I see. In response to that I had called the StepShooterPowerCommand before hand in order to test the effectiveness of the Stop Command